packages/runtime/native/CSS.ml

```diff
@@ -35,20 +35,22 @@ module Array = struct
   let flatten a = Array.concat (Array.to_list a)
 end
 
-let render_declaration rule =
+let render_declaration out rule =
   match rule with
   (* https://emotion.sh/docs/labels should be ignored on the rendering *)
-  | Rule.Declaration ("label", _value) -> None
+  | Rule.Declaration ("label", _value) -> ()
   | Rule.Declaration (property, value) ->
-    Some (Printf.sprintf "%s: %s;" property value)
-  | _ -> None
-
-let render_declarations rules =
-  rules
-  |> Array.map ~f:Autoprefixer.prefix
-  |> Array.flatten
-  |> Array.filter_map ~f:render_declaration
-  |> Array.join ~sep:" "
+    Buffer.add_string out property;
+    Buffer.add_string out ": ";
+    Buffer.add_string out value;
+    Buffer.add_char out ';';
+    Buffer.add_char out ' '
+  | _ -> ()
+
+let render_declarations out rules =
+  let ls = rules |> Array.map ~f:Autoprefixer.prefix |> Array.flatten in
```

Comment:
I have seen "ls" in a lot of places and I'm rather confused, could you make a better name?